cli-wrap/src/main/resources/cli/da_cli_script.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
 import argparse
 import requests
+requests.packages.urllib3.disable_warnings()
```

Comment:
it suppresses same warnings - http://stackoverflow.com/questions/27981545/suppress-insecurerequestwarning-unverified-https-request-is-being-made-in-pytho